docs/source/modules/socrates.rst

```diff
@@ -0,0 +1,209 @@
+
+SOCRATES Radiation Scheme Interface
+===================================
+
+Summary
+-------
+.. This summary is modified from Stephen Thomson's P/R for Socrates: <https://github.com/ExeClim/Isca/pull/61>
+
+SOCRATES (Suite Of Community RAdiative Transfer codes based on Edwards and Slingo) is the radiation scheme used by UK Met Office for Earth and planetary science [MannersEtAl2015]_, which has many significant advantages over RRTM, notably its flexibility in terms of atmospheric composition and the spectral properties of the radiation scheme (e.g. number of bands, etc).
+
+* The code used to integrate Socrates into Isca is contained within the folder ``src/atmos_params/socrates/interface``.
+* The Socrates source code itself is **NOT** packed within this Isca repository, and **NEW** users will need to download it from the `Met Office Science Repository <https://code.metoffice.gov.uk/trac/socrates>`_, and put it under ``src/atmos_params/socrates/src/trunk``. Detailed instructions on how to do this are included in the `README.md <https://github.com/ExeClim/Isca/blob/master/exp/test_cases/socrates_test/README.md>`_ for the Socrates test-case: ``exp/test_cases/socrates_test/README.md``.
```

Comment:
One quick request - there are two options for pointing to the socrates code. One is to put it under `src/atmos_params/socrates/src/trunk` as you say. The other is to set the environment variable `GFDL_SOC` equal to the directory where the socrates code is placed. Could you perhaps add this option to this section? So instead of 
", and put it under `src/atmos_params/socrates/src/trunk`" you could have:
". Users can then either choose to put the Socrates code within the directory `src/atmos_params/socrates/src/trunk`, or they can set the bash environment variable `GFDL_SOC` equal to the location of the source code for Socrates."

Reply:
Stephen, thank you very much for pointing out this option. I have updated the docs.